docs/install-docker.md

```diff
@@ -242,28 +294,7 @@ make init-docker
 make compose
 ```
 
-### Step 5: Update hosts
```

Comment:
I am not sure what is going on but your doc is out of date. This is adding changes that were removed from previous PR